stanford-earth/SE3_BLT
# SITES-8: Add travis enc file to pull from production

# READY FOR REVIEW

# Summary
- https://github.com/stanford-earth/SE3_BLT/pull/61 was missing a file that Travis needed to communicate with Acquia.  This pull request stores that file.

# Needed By
- Whenever you want Travis to start working, so that tests and be written and run during the pull request review process.

# Steps to Test
- You can see from the Travis build for this PR, that it has already gone beyond the step where it got stuck, namely finding travis.tar.enc: https://travis-ci.org/stanford-earth/SE3_BLT/builds/270216200.

# Affected versions or modules
- This is an addendum to: https://github.com/stanford-earth/SE3_BLT/pull/61

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-8
